.gitignore

```diff
@@ -4,7 +4,10 @@ config/nginx.conf
 config/skylight.yml
 
 tmp/
-logs/
+/log/
```

Comment:
I think an old gitignore got committed? I'll investigate, making a note for myself here.